cmd/sourced/cmd/refresh.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/src-d/sourced-ce/cmd/sourced/compose"
+)
+
+type refreshCmd struct {
+	Command `name:"refresh" short-description:"Update current installation according to the active docker compose file" long-description:"Update current installation according to the active docker compose file."`
```

Comment:
I understood that this command is not recreating volumes with charts, repos, and metadata, so I'd confirm it in the long-description, because `Update current installation` does not necessarily mean that volumes will be kept.

Reply:
please propose a better description.
it does not recreate volumes. it updates containers but all the data is preserved.